### ISSUE TITLE
Validation dataset inqitry

### ISSUE THREAD

**Issue by codingcode111** [newhyun00/paip2019_code#1]
Hi Newhyun,

Thank you so much for sharing the code. However, can I know exactly how did you divided the data; did you use the 10 validation data (XML, SVS) that are available currently in the PAIP  2019 Challenge, or you split the training data into training and validation set. Your reply and help would be highly appreciated. 

Kind regards,
